README.md

```diff
@@ -99,4 +99,49 @@ $ docker-compose run bintest
 * [mruby-eso-research](https://github.com/hone/mruby-eso-research) - an app for managing crafting research in Elder Scrolls Online. It uses YAML as the data store.
 
 ## mruby-cli Development
-This app is built as a mruby-cli app, so you just need to run: `docker-compose run compile` and find the binaries in the appropriate directories.
+
+### Compile the mruby-cli binaries
+
+This app is built as a `mruby-cli` app, so you just need to run:
+
+```
+docker-compose run compile
+```
+
+and find the binaries in the appropriate directories.
+
+#### Common error
+
+If rather than typing `docker-compose run compile`, you type `rake compile`, you could get on a nix OS the following error:
```

Comment:
This happens because users _must_* compile their binary in the mruby-cli docker container.

* Actually only if they want to use the cross-build configuration we generate.

Just using `rake compile` will not work because the main `Build` is designed to compile on a 64-bit Linux Host.